solidity/contracts/converter/types/standard-pool/StandardPoolConverter.sol

```diff
@@ -1286,6 +1283,25 @@ contract StandardPoolConverter is
         return _amount.mul(_reserveBalance) / _supply;
     }
 
+    /**
+     * @dev transfers funds held by the contract and sends them to an account
+     *
+     * @param token ERC20 token contract address
+     * @param to account to receive the new amount
+     * @param amount amount to withdraw
+     */
+    function safeTransfer(
```

Comment:
same comment as the one in the ConverterBase

Reply:
👍